incubator/metricbeat/templates/daemonset-map.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
you can remove this line

Reply:
Would you like me to remove the `---` header from all yaml files?

Reply:
why do you need that?

Reply:
just to make similar to other charts

Reply:
Thanks, I have removed. I had just converted this chart from a single config file so they were present from my source and hung around :)